source/_integrations/rmvtransport.markdown

```diff
@@ -87,8 +87,6 @@ The example below shows a full configuration with three sensors that showcase th
 # Example configuration.yaml entry
 sensor:
   - platform: rmvtransport
-    scan_interval: 120
```

Comment:
Since this is a platform integration, `scan_interval` is correct. This is handled by Home Assistant in this case, not the rmvtransport integration.

Why do you think it is invalid?